lib/ansible/modules/hostname.py

```diff
@@ -20,7 +20,7 @@
 description:
     - Set system's hostname, supports most OSs/Distributions, including those using systemd.
     - Note, this module does *NOT* modify C(/etc/hosts). You need to modify it yourself using other modules like template or replace.
-    - Windows, HP-UX and AIX are not currently supported.
+    - Windows, MacOS, HP-UX and AIX are not currently supported.
```

Comment:
I have a PR (#54439) that adds macOS support but no one seems interested in it. `¯\_(ツ)_/¯`
```suggestion
    - Windows, macOS, HP-UX and AIX are not currently supported.
```